workers/generic-worker/usage_multiuser.go

```diff
@@ -18,7 +18,10 @@ func headlessTasksUsage() string {
 func runTasksAsCurrentUserUsage() string {
 	return `
           runTasksAsCurrentUser             If true, users will still be created for tasks, but
-                                            tasks will be executed as the current OS user. [default: false]`
+                                            tasks will be executed as the current OS user.
+                                            Requires scope generic-worker:run-tasks-as-current-user:<provisionerID>/<workerType>.
+                                            Tasks submitted without this scope will be resolved
+                                            as exception/malformed-payload. [default: false]`
```

Comment:
I think we can lose `runTasksAsCurrentUserUsage` entirely since it is no longer a worker feature, but a task feature instead.

Reply:
Yep this is what I'm working on 👍🏻